manifests/nginx.pp

```diff
@@ -11,6 +11,23 @@
   $url_prefix = hiera("uber::config::url_prefix"),
 ) {
 
+  $ssl_crt_filename = "${nginx::params::conf_dir}/ssl-certificate.crt"
+  $ssl_key_filename = "${nginx::params::conf_dir}/ssl-certificate.key"
+
+  ensure_resource('file', $ssl_crt_filename, {
+    owner  => $nginx::params::daemon_user,
+    mode   => '0444',
+    source => $ssl_crt,
+    notify => Service["nginx"],
+  })
+
+  ensure_resource('file', $ssl_key_filename, {
+    owner  => $nginx::params::daemon_user,
+    mode   => '0440',
+    source => $ssl_key,
+    notify => Service["nginx"],
+  })
+
```

Comment:
we have to create our own keys now, the puppet module won't do it for us.  which is annoying, but for the best